bodhi/server/models/models.py

```diff
@@ -1022,15 +1022,19 @@ def get_cvestring(self):
     def get_bug_karma(self, bug):
         good, bad, seen = 0, 0, set()
         for comment in reversed(self.comments):
-            if comment.user.name in seen:
-                continue
-            seen.add(comment.user.name)
-            for feedback in comment.bug_feedback:
-                if feedback.bug == bug:
-                    if feedback.karma > 0:
-                        good += 1
-                    elif feedback.karma < 0:
-                        bad += 1
+            if comment.user:
+                if (comment.user.name == u'bodhi' and
+                        ('New build' in comment.text or 'Removed build' in comment.text)):
+                    break
+                if comment.user.name in seen:
+                    continue
```

Comment:
This line is missing test coverage, but it seems like was a pre-existing line so I don't think we have to add coverage for it unless you want bonus points ☺